Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -37,6 +37,8 @@ RWMol *cleanup(const RWMol &mol, const CleanupParameters &params) {
   RWMol *reionized = MolStandardize::reionize(normalized.get(), params);
   MolOps::assignStereochemistry(*reionized);
 
+  reionized->updateProps(m);
```

Comment:
I think it's worth putting a comment in front of this explaining what it's doing.